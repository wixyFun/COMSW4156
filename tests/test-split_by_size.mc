

int main()
{
  file myfile;
  file outputfilelist;

  myfile = open("bigbang.txt");
  /*split_by_size(myfile, 500);*/

  outputfilelist = split_by_quant(myfile, 10);

  return 0;
}
