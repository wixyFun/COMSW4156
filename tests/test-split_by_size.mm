

int main()
{
  file myfile;
  file outputfilelist;
  string test;

  myfile = open("bigbang.txt", "r");
  split_by_size(myfile, 500);

  /*outputfilelist = split_by_quant(myfile, 10);*/

  test = "hello world";
  printstring(test);

  return 0;
}
