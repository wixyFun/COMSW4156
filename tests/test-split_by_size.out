Number of characters is 4698
sizeFile is 4698
Number of chunks is 9
hello world
